examples/simple/src/dataProvider.tsx

```diff
@@ -17,6 +18,8 @@ const dataProvider = withLifecycleCallbacks(fakeRestProvider(data, true), [
             await dp.deleteMany('comments', {
                 ids: comments.map(comment => comment.id),
             });
+            // The queryClient would be unaware of the deleted comments without this.
+            queryClient.invalidateQueries({ queryKey: ['comments'] });
```

Comment:
the `useDeleteMany` should take care of this, otherwise that's a bug

Reply:
This is a lifecycle callback. The `useDelete` hook was called on the `post` resource. It doesn't even know about `comments`